.github/workflows/abstractions-php.yml

```diff
@@ -0,0 +1,28 @@
+name: PHP abstractions
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    paths: ['abstractions/php/**', '.github/workflows/**']
+  pull_request:
+    paths: ['abstractions/php/**', '.github/workflows/**']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      relativePath: ./abstractions/php
+    steps:
+      - uses: actions/checkout@v2.4.0
+        with:
+          php-version: '^7.4'
```

Comment:
are you sure this parameter goes with that action? it's not documented anywhere https://github.com/actions/checkout
did you mean to use this action? https://github.com/shivammathur/setup-php

Reply:
Thanks for catching this. Updated the PR.